qemu/tests/cfg/migrate_guest_with_iommu.cfg

```diff
@@ -0,0 +1,35 @@
+- migrate_guest_with_iommu:
+    virt_test_type = qemu
+    only q35
```

Comment:
`q35` only for x86, this line means the case will only test on x86 with q35.

Reply:
> `q35` only for x86, this line means the case will only test on x86 with q35.
Yes, this is the original intention of my design.